articles/discipline-flow.md

```diff
@@ -0,0 +1,62 @@
+---
+authorName: Stephan Janisch
+authorGithubUsername: sjanisch
+title: Discipline Flow
+---
+
+# {{page.title}}
+Product conception, development and delivery is a highly interdisciplinary team endeavour. Anybody who has ever participated in a more or less complete product lifecycle knows that not only the languages each discipline use are quite different, but also that the results of each discipline may sometimes do not fit to each other in the intended way.
+
+The Discipline Flow tackles this by relating the engineering disciplines in a general product lifecycle context. Its core use case is to get a comprehensive overview of disciplines and flows required for product conception, development and delivery - from user needs over business goals to the product as delivered by operations to the final user.
+
+## The Basic Model
+The very first step of any kind of meaningful product development must take the user as well as the business goals into account and by this means allow for continuous delivery of product increments. Relations between the Business (BUS), Customer Experience (CX), User Experience (UX), Requirements (REQ), Architecture (ARCH), Implementation (IMPL) and Operations (OPS) define the basic flow model to achieve this.  
+
+![Discipline Flow](./discipline-flow/dflow-sketch.jpg)
+
+The disciplines work on discipline-specific essentials: 
```

Comment:
"Disciplines" as in roles or titles? or people? or activities?
Discipline confuses me here...

Reply:
as in "software engineering disciplines"

Reply:
Das wäre dann wahrscheinlich eher "Each discipline works on ...". Oder würdest Du für das deutsche Wort "Disziplin" (im Sinne von Engineering Disziplin) eine andere englische Übersetzung nehmen?

Reply:
Ich finde "activity" besser... aber ist meine subjektive meinung.

Reply:
ok, danke. Ich lass mir das nochmal durch den Kopf gehen